qcodes/instrument/base.py

```diff
@@ -628,3 +628,12 @@ def __getstate__(self):
             'were trying to use a local instrument (defined with '
             'server_name=None) in a background Loop. Local instruments can '
             'only be used in Loops with background=False.')
+
+    def validate_status(self, verbose=0):
+        """ Validate the values of all gettable parameters """
```

Comment:
The `verbose` parameter is an int on purpose. In general I use the convetion: 0: no output, 1: normal output, >=2: used for debugging, output not stable